### DISCUSSION TITLE
Welcome to scholarly Discussions!

### DISCUSSION THREAD

**Discussion by ipeirotis** [scholarly-python-package/scholarly#223] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hp0404** [comment]
Hi there! J

ust wanted to ask if anyone could share examples of how they set up pipelines using scholarly? (Just to get a sense of some good practices or conventions - while figuring out how to avoid getting blocked by google).  
Also, what kind of proxy services one could recommend?
